gateway/api/v1/serializers.py

```diff
@@ -2,9 +2,11 @@
 Serializers api for V1.
 """
 
+import json
 from rest_framework.serializers import ValidationError
 from api import serializers
 from api.models import Provider
+from main import settings
```

Comment:
I know that we are not using this in all the places (I'm changing it in the PR's that I have opened) but the standard way to access to the settings in django is using:

```py
from django.conf import settings
```

https://docs.djangoproject.com/en/5.0/topics/settings/#using-settings-in-python-code